react/DepartmentHeader.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component, Fragment } from 'react'
```

Comment:
I think you don't need this.

Reply:
Without this is not possible to manage the top css property of the `MainCategories`. In other words, when there's no banner the `MainCategories` component will be partially covered by the `CategoryMenu`.